pages/groups/_groupId/_sessionId.vue

```diff
@@ -0,0 +1,404 @@
+                    >
+                      <v-btn
+                        color="green"
+                        outlined
+                        icon
+                        :disabled="user.present === true"
+                        @click="user.present = true"
+                      >
+                        <v-icon>mdi-account-check-outline</v-icon>
+                      </v-btn>
+                      <v-btn
+                        color="red"
+                        outlined
+                        icon
+                        class="ml-3 mr-2"
+                        :disabled="user.present === false"
+                        @click="user.present = false"
+                      >
+                        <v-icon>mdi-account-remove-outline</v-icon>
+                      </v-btn>
+                      <v-btn color="grey" icon>
+                        <v-icon>mdi-dots-horizontal</v-icon>
+                      </v-btn>
+                    </v-row>
+                  </v-list-item>
+                </v-card-actions>
+              </v-card>
+            </v-col>
+          </v-row>
+        </v-col>
+        <!-- QRCode Card -->
+        <v-col
+          v-if="sign_mode === 'qrcode'"
+          ref="qrcodeCanvasContainer"
+          v-resize="updateQrCodeThrottle"
+          cols="12"
+          md="6"
+          class="d-flex justify-center"
+          order="first"
+          order-md="0"
+        >
+          <canvas ref="qrcodeCanvas" class="rounded" style="border: 1px solid #0000001F; max-width: 100%;" />
+        </v-col>
+        <!-- Code Card -->
+        <v-col
+          v-if="sign_mode === 'code'"
+          cols="12"
+          md="6"
+          lg="4"
+          order="first"
+          order-md="0"
+        >
+          <v-card class="text-center py-8" outlined>
+            <div class="text-h4 mb-4">
+              Session Code
+            </div>
+            <div class="text-h2" style="letter-spacing: 10px !important;">
+              {{ session_code }}
+            </div>
+          </v-card>
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex'
+import debounce from 'lodash.debounce'
+import moment from 'moment'
+import QRCode from 'qrcode'
+
+function addZero (i) {
+  if (i < 10) { i = '0' + i }
+  return i
+}
+
+export default {
+  name: 'SessionPage',
+  filters: {
+    presentColor (value) {
+      return value !== null ? value ? '#4CAF5033' : '#F4433633' : ''
+    }
+  },
+  data () {
+    return {
+      users: [],
+      sign_mode: 'none',
+      session_code: '000000',
+      session_end: moment().add(10, 'hours').toDate(),
+      qr_code_data: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c',
+      qr_code_interval: null,
+      time_left: '00:00:00',
+      time_left_interval: null
```

Comment:
Please use `undefined` instead of `null`.
It's keep RAM allocation.

Reply:
if you want, its still set like a microsecond after